src/urllib3/connection.py

```diff
@@ -286,7 +288,7 @@ def request_chunked(
         self,
         method: str,
         url: str,
-        body: Union[None, HTTPBody, Tuple[Union[bytes, str]]] = None,
+        body: Union[None, _TYPE_HTTP_BODY, Tuple[Union[bytes, str]]] = None,
```

Comment:
Seeing this here has alerted me to a type issue, woo!

This type here should be `Optional[_TYPE_BODY]` and line 316 should be changed to `body = (to_bytes(body),)`. You'll have to import `to_bytes` above as well.

Reply:
Done.